src/grammar.coffee

```diff
@@ -353,6 +353,7 @@ grammar =
 
   Import: [
     o 'IMPORT Expression',                      -> new Import $2
+    o 'IMPORT Assignable FROM Expression',      -> new Import $4, $2
```

Comment:
I thought this was a destructured assignment, just like CS already supports (syntactically). Is it not?